src/ssr-carousel-dots.vue

```diff
@@ -5,10 +5,13 @@
 .ssr-carousel-dots
 	button.ssr-carousel-dot-button(
 		v-for='i in pages' :key='i'
-		:aria-label='`Page ${i}`'
+		:aria-label='`Go to page ${i}`'
 		:disabled='isDisabled(i)'
 		@click='$emit("goto", i - 1)')
 
+		//- visually hidden text that states the current slide is selected
+		span.visually-hidden(v-if='isDisabled(i)') Selected
+
```

Comment:
Could you check with the ADA folks you're working with if [`aria-current=true`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current) could be used instead of this? This is also mentioned as a suggestion on the MDN `button` docs: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#aria_state_information.

Reply:
I asked them in their ticket, and I'll let you know what they say. Added you as a watcher on the asana task here: https://app.asana.com/0/1202074446600011/1203557689732170/f